apps/odo/config/config.exs

```diff
@@ -0,0 +1,11 @@
+use Mix.Config
+
+config :odo,
+  collector: StreamingMetrics.PrometheusMetricCollector
+
+config :mime, :types, %{
```

Comment:
this `mime` block should be removed now as it will inherit from the top-level config file in the umbrella.

Reply:
Removed